src/sock.c

```diff
@@ -234,6 +234,25 @@ struct dnsquery_resourcerecord
 	struct dnsquery_srvrdata rdata;
 };
 
+void netbuf_add_32bitnum(unsigned char *, int, int *, unsigned int);
```

Comment:
Sorry, I don't understand what the problem these prototypes solve. Could you describe it?